### PR TITLE
Option polls

### PR DESCRIPTION
**Pull request by DeinFreund**
Polls now have multiple options instead of just Yes and No.

### CODE REVIEW

**Comment by Licho1** [comment]
Note that this requires changes ingame/chobby

**Comment by Licho1** [comment]
Note that this requires changes ingame/chobby

**Comment by Licho1** [comment]
Note that this requires changes ingame/chobby

**Comment by DeinFreund** [comment]
Yes, I'm waiting with this merge until @GoogleFrog is ready to use it in Chobby.

File: ZkLobbyServer/ServerBattle.cs [hunk 1]
```diff
@@ -762,7 +794,32 @@ private void discussionTimer_Elapsed(object sender, ElapsedEventArgs e)
             discussionTimer.Stop();
             var poll = new CommandPoll(this, false);
             poll.PollEnded += MapVoteEnded;
-            StartVote(new CmdMap(), null, "", MapVoteTime, poll);
+            var options = new List<PollOption>();
+            for (int i = 0; i < NumberOfMapChoices; i++)
+            {
+                Resource map;
+                if (i < NumberOfMapChoices / 2 && MinimalMapSupportLevel < MapSupportLevel.Featured)
```
[the remark this answers is not in the document]
Comment (DeinFreund) [reply ZkLobbyServer/ServerBattle.cs]: A server could be set to only allow Matchmaker maps. Imagine a 1v1 training server for example. In this case he should not show any featured maps.

**Pull request merged**